envbuilder-dogfood/main.tf

```diff
@@ -336,33 +339,49 @@ resource "docker_image" "envbuilder" {
   keep_locally  = true
 }
 
+locals {
+  cache_repo = "us-central1-docker.pkg.dev/coder-dogfood-v2/envbuilder-cache/coder-dogfood"
+  envbuilder_env = {
+    "CODER_AGENT_TOKEN" : coder_agent.dev.token,
+    "CODER_AGENT_URL" : data.coder_workspace.me.access_url,
```

Comment:
I suppose these could be set on the container only, but fine to keep here for convenience 👍🏻